src/scripts/tests/simplePawnMoves.spec.ts

```
@@ -0,0 +1,42 @@
+  });
+
+  test('Jump over', () => {
+    const board = new Board();
+    const piece = new Piece(false, new Vector2(0, 0));
+    board.addPiece(piece);
+    board.addPiece(new Piece(false, new Vector2(1, 1)));
+    board.addPiece(new Piece(false, new Vector2(0, 1)));
+
+    const moves = MoveCalculator.calculateMoves(board, piece);
+
+    assert.lengthOf(moves!, 2);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)